### DISCUSSION TITLE
Error object not visible in issue

### DISCUSSION THREAD

**Discussion by bartschriever** [getsentry/support#13] [Support]
Hi! I'm currently experiencing some issues in Sentry that I can't get any information from. Below are a number of images with data that I can't get anything from. I've searched everything but haven't found a solution yet. Do you have any idea how I can solve this?

The issue in Sentry
![image](https://user-images.githubusercontent.com/35870713/171845707-7005a3da-b742-4828-a20f-fecc85e11a7e.png)

The error object inside the issue
![Schermafbeelding 2022-06-03 om 13 51 09](https://user-images.githubusercontent.com/35870713/171848471-f4ee528e-27aa-44dd-83c7-a9e1688bfb0e.png)

As you can see all the values inside error are undefined. The response object is the most important for us when somethings happens. How does it come that config.data is visible but response.data only says data: [Object]

**Comment by souredoutlook** [comment]
Hi there,

Can you try increasing the value of the [normalizeDepth option](https://docs.sentry.io/platforms/javascript/configuration/options/#normalize-depth) from its default of 3 and let me know if this resolves your missing data issue?
